packages/backend/src/open_payments/payment/outgoing/worker.ts

```diff
@@ -35,8 +35,7 @@ export async function processPendingPayment(
 }
 
 // Fetch (and lock) a payment for work.
-// Exported for testing.
```

Comment:
Did we not test it or are we not testing it anymore or do we test somewhere else?

Reply:
This was never tested directly, the tests for `processNext` tested this